frontend/src/components/common/form/ParentSelect.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
Would it be helpful to have tests to validate the the clicks on parent and child options work as expected?

Reply:
On it